_posts/2020-04-01-cockpit-216.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Cockpit 216
+author: lis
+date: 2020-04-01
+tags: cockpit linux
+slug: cockpit-216
+category: release
+summary: Cockpit with improved SELinux page and last login banner
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 216.
+
+### SELinux: Automatic application of solutions that set booleans
+
+The SELinux page can now automatically apply solutions that set [SELinux booleans](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/selinux_users_and_administrators_guide/sect-security-enhanced_linux-working_with_selinux-booleans).
+
+![Apply this solution](/images/selinux-apply-this-solution.png)
+
+In addition, the command lines in solution details are now formatted properly.
+
+### Machines: Drop virsh backend support
+
+This release drops the virsh backend support from the machines page and
+switches over to using libvirt-dbus exclusively.  As a result, as of this
+release, cockpit-machines is no longer supported on Ubuntu 18.04.
+
+### Overview: New "last login" banner
+
+Similar to text logins at the console, or via SSH, Cockpit now displays a
+banner showing the last successful login.  If there have been failed login
+attempts, the number of attempts and the time of the most recent failed attempt
+will be shown as well.
+
+![Last login banner](/images/last-login-banner.png)
+
+A side-effect of this work is that Cockpit now correctly updates
+`/var/log/lastlog` (the record of last login times) and `btmp` (the
+record of failed login attempts).
+
+### Try it out
+
+Cockpit 216 is available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/216)
+ * [Fedora 31](https://bodhi.fedoraproject.org/updates/FEDORA-2020-TODO)
+ * [Fedora 32](https://bodhi.fedoraproject.org/updates/FEDORA-2020-TODO)
```

Comment:
```suggestion
 * [Fedora 32](https://bodhi.fedoraproject.org/updates/FEDORA-2020-TODO)
 
 *[SELinux]: security-enhanced Linux, a way to ensure better access control to resources
 *[virsh]: virtual machine shell, a utility to control virtual machines at the command line
 *[libvirt-dbus]: a way to control virtual machines over the common "d-bus" inter-program communication layer 
*[SSH]: secure shell
```

Added some definnitions. Trying to explain an API like d-bus in simpler English is difficult. I might've gotten it wrong. Please fix. :wink:

Reply:
It's "D-Bus", and "inter-process" communication, but otherwise looks  good, thanks!